### PR TITLE
feat: added notes page

### PR DESCRIPTION
**Pull request by tanishq20**
Added new notes page

- API call to add new notes
- Display Notes from backend
screencapture-localhost-3000-notes-2022-04-07-11_38_57

### CODE REVIEW

**Review by rahulcs754 — APPROVED** [review]
The code looks clean and good.
before push your code remove comment line and console log

File: src/context/notes/notesProvider.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+export const NotesProvider = ({ children }) => {
+  const defaultNotes = {
+    title: '',
+    label: '',
+    priority: '',
+    noteBody: '',
+    bgColor: '#21272d',
+    isPinned: false,
+    isEditing: false,
+    date: new Date().toLocaleDateString(),
```
Comment (coldpigli) [inline src/context/notes/notesProvider.js:15]: Nice that you are also showing the time and date of the note.

File: src/pages/notes/Notes.js [hunk 2]
```diff
@@ -1,7 +1,69 @@
...
+                      onClick={() => setToggleNewNoteCard((prev) => !prev)}
+                    >
+                      Create New Note
+                    </button>
+                  </div>
+                )}
+                {notes[0] && (
+                  <>
+                    <p className='category-heading'>My Notes</p>
+                    {notes?.map((noteDetails) => (
```
Comment (coldpigli) [inline src/pages/notes/Notes.js:46]: Good to see you have used optional chaining here

**Review by coldpigli — APPROVED** [review]
Have pointed out things I liked. Everything looks good.

**Pull request merged**